[Dev] Surface Magnetism + Spatial Processing example scene - improvement

cre8ivepark opened the pull request:
Overview
---
- Added transparent surface to make it easier to see the interaction with physical surfaces.
- Assigned WireframeBlue material to the example scene. (it was not assigned)
- [Updated landing page README for the discoverability](https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/Dev_Working_Branch_SurfaceMagnetism/README.md)
![2018-05-17 13_28_16-mixedrealitytoolkit-unity_readme md at dev_working_branch_surfacemagnetism cre](https://user-images.githubusercontent.com/13754172/40202234-31d1a038-59d6-11e8-9916-02aa43f7b51e.png)


Changes
---
- Updates to #2113

StephenHodgson commented on ProjectSettings/TagManager.asset:
Not sure we should be adding anything to the tag manager manually. 

When people import this example, tags are generally not included.

cre8ivepark commented on ProjectSettings/TagManager.asset:
If that's the case, I guess developers have to manually add the layer?

StephenHodgson commented on ProjectSettings/TagManager.asset:
indeed, or we can hard code it.

StephenHodgson commented on ProjectSettings/TagManager.asset:
It's related to #1443

cre8ivepark commented on ProjectSettings/TagManager.asset:
I just read #1443. In this case, I think we can use layer 31:Spatial Mapping and add instruction to use Configure > Apply Mixed Reality Project Settings.
Thank you!

![2018-05-17 12_49_47-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40200329-dc8cc4a4-59d0-11e8-8f23-05e0d06bdbbe.png)

![2018-05-17 12_49_40-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40200330-dca1a7fc-59d0-11e8-988c-9e0339ad3d71.png)

cre8ivepark commented:
- Removed TagManager.asset
- Updated README to use MRTK's Configure menu to add the layer 31:SpatialMapping
- Updated scene to use the layer 31:SpatialMapping

https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/may18_dev_SurfaceMagnetism/Assets/HoloToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md

davidkline-ms commented on Assets/MixedRealityToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md:
This is the default within the prefab. I would reword step 2 to say "make sure the Physics Layer is set to 31"

davidkline-ms commented on Assets/MixedRealityToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md:
"select the SpatialMapping layer."  they would not have created it

cre8ivepark commented on Assets/MixedRealityToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md:
Updated, Thanks!
